Dockerfile

```diff
@@ -1,4 +1,47 @@
-FROM sismics/ubuntu-jetty:11.0.14
+FROM ubuntu:22.04
+
+# Run Debian in non interactive mode
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install Sismics repository
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates software-properties-common curl gnupg tzdata
+RUN curl -fsSL https://www.sismics.com/pgp | apt-key add -
+# RUN add-apt-repository "deb [arch=amd64] https://nexus.sismics.com/repository/apt-bionic/ bionic main"
+
+# Configure settings
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+RUN ln -fs /usr/share/zoneinfo/Europe/Paris /etc/localtime
```

Comment:
What exactly does this mean for people not in the Paris timezone?

Reply:
I need @jendib's help in answering this. I am just combining 3 dockerfiles into one for now

Reply:
I don't think it matters in any way because it's already the case currently and nobody complained ;)